include/grpc/impl/channel_arg_names.h

```diff
@@ -51,11 +51,16 @@
  * server will close the connection. Int valued, milliseconds. INT_MAX means
  * unlimited. Defaults to INT_MAX. */
 #define GRPC_ARG_MAX_CONNECTION_IDLE_MS "grpc.max_connection_idle_ms"
-/** Maximum time that a channel may exist. Int valued, milliseconds.
- * INT_MAX means unlimited. Defaults to INT_MAX. */
+/** Maximum time in milliseconds that a channel may exist. Int valued, milliseconds.
+  * when time expires, the server sends a 'GOAWAY' frame to the client.
+  * New RPCs will not be accepted, but existing ones can continue untill they complete.
+  * If grpc.max_connection_age_ms is not set, active RPCs may be interrupted when the connection closes.
+  * Defaults to INT_MAX. */
 #define GRPC_ARG_MAX_CONNECTION_AGE_MS "grpc.max_connection_age_ms"
-/** Grace period after the channel reaches its max age. Int valued,
-   milliseconds. INT_MAX means unlimited. Defaults to INT_MAX. */
+/** Grace period after the channel reaches its max age. Int valued, milliseconds.
```

Comment:
Let's just say - 
"Grace period in milliseconds after connection reaches its max age for outstanding RPCs to complete. Int valued, defaults to INT_MAX (disabled)."